src/org/labkey/targetedms/view/precursorChromatogramsView.jsp

```diff
@@ -40,7 +40,7 @@
     </tr>
     <tr>
         <%
-            String fieldLabel = bean.getPeptideGroup().getSequenceId() != null ? "Protein" : "Group";
+            String fieldLabel = bean.getPeptideGroup().getProteins().isEmpty() ? "Group" : "Protein";
```

Comment:
can "group" and "protein" be constants somewhere?

Reply:
I consolidated this snippet of code so it's not copied in three JSPs. I didn't see a lot of value in a constant since no other code is acting on the value being displayed (this isn't inherently linked with the name of the <protein> element in the XML files, for example, and the casing is different regardless.